Maddori.Apple/Maddori.Apple/Screen/InProgress/InProgressViewController.swift

```diff
@@ -0,0 +1,197 @@
+//
+//  InProgressViewController.swift
+//  Maddori.Apple
+//
+//  Created by 이성민 on 2022/10/24.
+//
+
+import UIKit
+
+import SnapKit
+
+final class InProgressViewController: BaseViewController {
+    
+    private var keywordData = Keyword.mockData
+    private let currentRetrospective = "진저"
```

Comment:
p4;

```suggestion
    private let currentRetrospectiveUser = "진저"
```
아래쪽에서 `label.text = currentRetrospective + TextLiteral.InProgressViewControllerSubTitleLabel`이 사용된 부분을 보니까, 어떤 변수인지 헷갈려서 확인해 봤는데 **현재회고유저** 의 느낌으로 User를 붙이는 건 어떨까요 ??

Reply:
오오 그러게요 끝에만 user를 붙이면 되겠군요 반영했습니다!
[75ae654](https://github.com/DeveloperAcademy-POSTECH/MacC-Team-Maddori.Apple/pull/59/commits/75ae65462af86b8550d856734fe701d324aadd31)